### ISSUE TITLE
Phoron grenades do nothing

### ISSUE THREAD

**Issue by FenodyreeAv** [PolarisSS13/Polaris#5594]
#### Brief description of the issue
Grenades full of liquid phoron do nothing.

#### What you expected to happen
A big fire.

#### What actually happened
There was no phoron gas, liquid, or anything flammable produced.

#### Steps to reproduce
Fill a 600u grenade with phoron, detonate it, and set it on fire.

**Comment by Atermonera** [comment]
When a grenade goes off, it just mixes the two beakers and lets any reactions react. If you want phoron gas, then you need either the reagents to produce smoke, which will bear the phoron into the air, or have a reaction that produces gaseous phoron, which would probably require a machine that can transfer the results of such operation to an atmos pipe, which takes code.

**Comment by Atermonera** [comment]
Is it spreading liquid phoron across the floor? Or just clicking as it mixes the beakers?

**Comment by Atermonera** [comment]
Right, so the issue is more that the grenade isn't dispersing its liquid reagents like, say, a foam grenade does, and leaving behind puddles of phoron like dragging an opened weldertank leaves fuel and probably being less slippery. I can see how this'd be desirable for some reactions/effects, but not for others, depending on the specific effects of the reaction.
Whoever handles this, I'd recommend adding a "chemical dispersal" mode to grenades that can be toggled on while constructing the grenade, and upon detonation I think you can steal code from flamethrowers, iirc they spread their fuel and then ignite.
